cucumber-expressions/ruby/spec/cucumber/cucumber_expressions/cucumber_expression_generator_spec.rb

```diff
@@ -162,6 +162,58 @@ class Currency
         expect(expressions[1].source).to eq("{zero-or-more} {zero-or-more} {zero-or-more}")
       end
 
+      context "does not suggest parameter when match is" do
+        let!(:direction_parameter_type) {
```

Comment:
not of huge importance, but do these need to be let bangs? It'll cause some issues for local runs potentially. It'll also annoy rubocop.

These aren't blockers. And if needs to remain, I'm :cool: with that. But if we can change them just to `let` that would be better

Reply:
It's done :)

For the let! part, it was useful in the previous version to be able to use let(:force_full_word) and override it in sub-context but that does not make sense anymore indeed :)